packages/sdk/src/wallet-utils/validate-tx.js

```diff
@@ -0,0 +1,17 @@
+import {
+  encodeTransactionPayload as encodeInsideMessage,
+  encodeTransactionEnvelope as encodeOutsideMessage,
+} from "../encode/encode.js"
+import {invariant} from "@onflow/util-invariant"
+import {TRANSACTION} from "../interaction/interaction"
+
+export const validateSignableTransaction = signable => {
+  invariant(
+    signable.interaction.tag === TRANSACTION,
+    "Signable payload must be transaction"
+  )
```

Comment:
As discussed lets pull this out, as the interaction wont be included here in the future

Reply:
Pulled and refactored